packages/std/src/iterator.rs

```diff
@@ -1,8 +1,13 @@
 use crate::errors::StdError;
 use std::convert::TryFrom;
 
+/// A Key-Value pair, returned from our iterators
+pub type Pair<V = Vec<u8>> = (Vec<u8>, V);
+
 /// KV is a Key-Value pair, returned from our iterators
-pub type KV<T = Vec<u8>> = (Vec<u8>, T);
+#[deprecated(since = "0.14.0", note = "Renamed to Pair")]
+#[allow(clippy::upper_case_acronyms)]
```

Comment:
That's a rule that we might consider as standard. Uppercase acronyms are a pleonasm, by the way.

Reply:
There is this never ending debate whether or not acronyms should get special treatment in camelCase/PascalCase variable names. There are some ecosystems that say they should (e.g. Go and [Swift](https://github.com/raywenderlich/swift-style-guide/issues/101#issuecomment-713268522)) and others that say the acronym is less important than the word separation (e.g. the [Google Java style guide](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)). There are also middle ways saying that acronyms in general are to be treaded as regular words, but short acronyms get the same casing (Especially Microsoft; e.g. MyClassIO vs. MyClassJson, [see C#](https://stackoverflow.com/a/2109187/2013738)).

I'm I huge supporter of the model that gives priority to word separation and treats acronyms as any other word. Because if you say HTTPRequest, you also have to say HTTPJSONAPI. So I am more than happy Rust is following the path described in the Google Java style guide and manifests this via linting.

In this case we could have converted KV to Kv, but this is neither a word nor an acronym on its own and pair seems to be the nicer choice.